ext/pdo_firebird/php_pdo_firebird_int.h

```diff
@@ -136,6 +135,15 @@ extern void _firebird_error(pdo_dbh_t *dbh, pdo_stmt_t *stmt, const char *state,
 #define firebird_error_with_info(d,e,el,m,ml) _firebird_error(d, NULL, e, el, m, ml)
 #define firebird_error_stmt_with_info(s,e,el,m,ml) _firebird_error(s->dbh, s, e, el, m, ml)
 
+#define FB_TXN_RETAIN 1
+#define FB_TXN_RELEASE 0
+
+extern bool _firebird_commit_transaction(pdo_dbh_t *dbh, bool retain);
+#define firebird_commit_transaction(d,r) _firebird_commit_transaction(d, r)
```

Comment:
```suggestion
extern bool firebird_commit_transaction(pdo_dbh_t *dbh, bool retain);
```
Why not just this?

Reply:
I misunderstood what the naming convention should be and it became complicated.